### ISSUE TITLE
Can not upload App bundles .aab chplay

### ISSUE THREAD

**Issue by Doan-RIOT** [RocketChat/Rocket.Chat.ReactNative#4489]
<img width="1171" alt="Screen Shot 2022-09-04 at 14 35 14" src="https://user-images.githubusercontent.com/46195209/188302849-ff2dfc9b-d2ac-4751-b922-c94aab1a2ef8.png">

### Additional context:

Hope everybody help please!

**Comment by Doan-RIOT** [comment]
<img width="1197" alt="Screen Shot 2022-09-05 at 08 56 24" src="https://user-images.githubusercontent.com/46195209/188345361-3b334bb0-79c4-4444-aa29-0b344b5240db.png">
if set targetSdkVersion = 31

**Comment by titomiky** [comment]
Same...

**Comment by diegolmello** [comment]
Hey, thanks for reporting.
It's a requirement for new apps already and it's going to be required for updates on November https://developer.android.com/google/play/requirements/target-sdk

I'm adding to the next sprint so we can fix it asap.
If you find the `missing export` reason, please open a PR.

**Comment by Doan-RIOT** [comment]
> Hey, thanks for reporting. It's a requirement for new apps already and it's going to be required for updates on November https://developer.android.com/google/play/requirements/target-sdk
> 
> I'm adding to the next sprint so we can fix it asap. If you find the `missing export` reason, please open a PR.

Do you have any quick solution to fix this error?

**Comment by diegolmello** [comment]
Not really. I tried targeting 31 on #4316, but it was taking too long.

**Comment by Doan-RIOT** [comment]
> Not really. I tried targeting 31 on #4316, but it was taking too long.

I tried adding android:exported to the declarations in the manifest but it didn't work :(

**Comment by diegolmello** [comment]
@Doan-RIOT yeah, same here. That's why I created a separate task to be able to deliver the upgrade of RN.
This "exported" thing didn't make any sense to me as everything is all set, but let's take a look.

**Comment by Doan-RIOT** [comment]
Do you have any solution for this error. I've been trying to find a solution to this problem for a while now, but I'm still stuck. Please give me a solution so that I can continue to do my work!!!. @diegolmello

**Comment by Doan-RIOT** [comment]
Thanks very much @diegolmello . I have solved the problem. Will close this issue.
